src/main/java/teammates/storage/sqlapi/UsersDb.java

```diff
@@ -140,6 +142,30 @@ public Student getStudentByGoogleId(String courseId, String googleId) {
         return HibernateUtil.createQuery(cr).getResultStream().findFirst().orElse(null);
     }
 
+    /**
+     * Gets all instructors and students by {@code googleId}.
+     */
+    public List<User> getAllUsersByGoogleId(String googleId) {
+        CriteriaBuilder cb = HibernateUtil.getCriteriaBuilder();
+        CriteriaQuery<Instructor> instructorCr = cb.createQuery(Instructor.class);
+        Root<Instructor> instructorRoot = instructorCr.from(Instructor.class);
+        Join<Instructor, Account> instructorAccountsJoin = instructorRoot.join("account");
+
+        instructorCr.select(instructorRoot).where(cb.equal(instructorAccountsJoin.get("googleId"), googleId));
+
+        List<Instructor> instructors = HibernateUtil.createQuery(instructorCr).getResultList();
+
+        CriteriaQuery<Student> studentCr = cb.createQuery(Student.class);
+        Root<Student> studentRoot = studentCr.from(Student.class);
+        Join<Student, Account> studentAccountsJoin = studentRoot.join("account");
+
+        studentCr.select(studentRoot).where(cb.equal(studentAccountsJoin.get("googleId"), googleId));
+
+        List<Student> students = HibernateUtil.createQuery(studentCr).getResultList();
+
+        return Stream.concat(instructors.stream(), students.stream()).collect(Collectors.toList());
```

Comment:
```suggestion
        CriteriaBuilder cb = HibernateUtil.getCriteriaBuilder();
        CriteriaQuery<User> usersCr = cb.createQuery(User.class);
        Root<User> usersRoot = usersCr.from(User.class);
        Join<Users, Account> accountsJoin = usersRoot.join("account");

        usersCr.select(usersRoot).where(cb.equal(accountsJoin.get("googleId"), googleId));

        return HibernateUtil.createQuery(usersCr).getResultList();
```

Reply:
Ah! Wanted to do this but wasn't sure if it will remove records from Instructors and Students table. I suppose it does 👍🏻 Thanks for catching this!